vnext/react-native.config.js

```diff
@@ -2,16 +2,77 @@
 require('source-map-support').install();
 
 const cli = require('@react-native-windows/cli');
+const { execSync } = require('child_process');
+const path = require('path');
+
+const cliDir = path.dirname(require.resolve('@react-native-community/cli/package.json'));
+const execaPath = require.resolve('execa', { paths: [cliDir] });
+const execa = require(execaPath);
 
 module.exports = {
-    // **** This section defined commands and options on how to provide the Windows platform to external applications
-    commands: cli.commands,
-    platforms: {
-      windows: {
-        linkConfig: () => null,
-        projectConfig: cli.projectConfig,
-        dependencyConfig: cli.dependencyConfig,
-        npmPackageName: 'react-native-windows',
-      },
+  // **** This section defined commands and options on how to provide the Windows platform to external applications
+  commands: cli.commands,
+  platforms: {
+    windows: {
+      linkConfig: () => null,
+      projectConfig: cli.projectConfig,
+      dependencyConfig: cli.dependencyConfig,
+      npmPackageName: 'react-native-windows',
     },
-  };
+  }
+};
+
+// All our health checks are windows only...
+if (process.platform === 'win32') {
+  const rnwDepScriptPath = path.resolve(__dirname, 'Scripts/rnw-dependencies.ps1');
+  try {
+    const rnwDeps = execSync(`powershell ${rnwDepScriptPath} -NoPrompt -ListChecks`);
```

Comment:
Do we need to pass the ExecutionPolicy flags here to deal with the powershell default of not allowing any file comming from web (or git) to run.
Developers not used to windows will likely have never heard of powershell let alone know how to disable the policy.

Reply:
We should definatley pass --noprofile to avoid users customized powershell extensions and plugins to affect the output of the tool.